expo-maps/android/src/main/java/expo/modules/maps/ExpoGoogleMapsModule.kt

```diff
@@ -4,6 +4,12 @@ import expo.modules.core.interfaces.services.UIManager
 import expo.modules.kotlin.modules.Module
 import expo.modules.kotlin.modules.ModuleDefinition
 
+enum class MapType(val value: String) {
+  normal("normal"),
+  hybrid("hybrid"),
+  satellite("satellite"),
+  terrain("terrain")
+}
```

Comment:
Popular convention is to name enum values in capital letters but this is only suggestion. We can extract this enum class to its own file.